### ISSUE TITLE
False Positive: Layout/EmptyLinesAroundArguments With HEREDOCs

### ISSUE THREAD

**Issue by jfelchner** [bbatsov/rubocop#5355]
HEREDOCs are used frequently for complex cases with multiline strings.  As such, there are frequently cases where you might want to have empty lines as the first/last line in a HEREDOC.  I think that the `Layout/EmptyLinesAroundArguments` missed this use case when it was originally developed.

--------

## Expected behavior

HEREDOCs should be allowed to have empty lines at their start/end

## Actual behavior

HEREDOCs throw an error if they contain an empty line as their first/last line.

## Steps to reproduce the problem

```ruby
# baz.rb
class Foo
  def bar
    inject_into_file 'config/routes.rb',
                     <<~HEREDOC,
                       some_content

                     HEREDOC
                     after: "some_line_content\n"
  end
end
```

```ruby
# .rubocop.yml
AllCops:
  TargetRubyVersion: '2.4.3'
```

    > rubocop --only='Layout/EmptyLinesAroundArguments' baz.rb

## RuboCop version

```
$ rubocop -V
0.52.1 (using Parser 2.4.0.2, running on ruby 2.4.3 x86_64-darwin16)
```

**Comment by garettarrowood** [comment]
Since the last version bump, there was one more sweep through this Cop. I believe this scenario is resolved by #5340 .

**Comment by jfelchner** [comment]
@garettarrowood awesome!  I'll close this for now then.
